playground/src/PlaygroundView.tsx

```diff
@@ -135,7 +135,7 @@ export class PlaygroundView extends React.Component<IPlaygroundViewProps, IPlayg
         </FlexColDiv>
 
         <FlexRowDiv className='playground-footer' style={ footerStyle }>
-          &copy; 2018 Microsoft
+          &copy; 2019 Microsoft
```

Comment:
Thanks for fixing this!  So we don't have to fix it again next year, maybe we should use `Date.getFullYear()`?

Reply:
I think it's not good idea, because user date can be incorrect. Or this not matter?